src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Export/ExportJobTaskTests.cs

```diff
@@ -986,6 +987,31 @@ public async Task GivenAnExportJobToResumeWithTheTypeParameter_WhenExecuted_Then
             Assert.Equal(OperationStatus.Completed, _exportJobRecord.Status);
         }
 
+        [Fact]
```

Comment:
Should we create/adapt a test where NoGroupCount is expected to be zero?

Reply:
Done